### ISSUE TITLE
list dividers role="heading" fails w3c validator

### ISSUE THREAD

**Issue by frequent** [jquery/jquery-mobile#5268]
I'm using this:

    <li data-role="list-divider" role="heading" class="ui-li ui-li-divider ui-body-c ui-corner-top">
         <span class="ui-divider-text">#variables.li_header#</span>
    </li>

which is the same list-divider syntax as in the docs. Running this through the w3c validtor throws  `Bad value heading for attribute role on element li.`

Removing the `role="heading" fixes it, but I guess it's there for a reason. Any way to make this valid?

**Comment by frequent** [comment]
@Ruffio - Works here, too. Will close. If I ever stumble back on the issue I will repost. Thanks for coming back to this.
